app/javascript/packs/application.js

```diff
@@ -17,4 +17,5 @@ require("../src/index")
 // or the `imagePath` JavaScript helper below.
 //
 const images = require.context("../images", true)
+// eslint-disable-next-line no-unused-vars
```

Comment:
Are we putting this here for a specific reason? It looks like the kind of thing that should go at the top of a file to help distinguish itself as something other than a regular comment in code.